pyseq.py

```diff
@@ -228,12 +244,12 @@ def __repr__(self):
         return '<pyseq.Sequence "%s">' % str(self)
 
     def __getattr__(self, key):
-        return getattr(self[0], key, None)
```

Comment:
I'm open to debate on this, but I don't think calling getattr should return None for an invalid key.  This actually, caused me a pretty big headache when I was trying to unit test something a while back.